src/coordinator/index.js

```diff
@@ -102,6 +174,31 @@ class Coordinator {
       .then((orgs) => orgs.map((org) => toCamelCase(org)));
   }
 
+  /**
+   * Gets a user's list of favorited applications
+   *
+   * API Endpoint: '/applications/favorites'
+   * Method: GET
+   *
+   *
+   * @returns {Promise}
+   * @fulfill {ContxtUserFavoriteApplication[]} A list of favorited applications
+   * @reject {Error}
+   *
+   * @example
+   * contxtSdk.coordinator
+   *   .getFavoriteApplications()
+   *   .then((favoriteApplications) => console.log(favoriteApplications))
+   *   .catch((err) => console.log(err));
+   */
+  getFavoriteApplications() {
+    return this._request
+      .get(`${this._baseUrl}/applications/favorites`)
+      .then((favoriteApps) =>
+        favoriteApps.map((favoriteApp) => toCamelCase(favoriteApp))
```

Comment:
You can actually just feed `favoriteApps` into `toCamelCase`! (https://github.com/ndustrialio/contxt-sdk-js/blob/master/src/utils/objects/createCaseChangeFn.js#L59)

Reply:
Done 👍